### PR TITLE
fix(graphiql): use correct endpoint url for GraphiQL

### LINKED ISSUE #192: graphiql endpointURL should be configurable
In https://github.com/GrappleGQL/wagtail-grapple/blob/main/grapple/urls.py#L14 the graphql-endpoint it set statically to `/graphql`. The default configuration used in doc is exposing `/api/graphql`. This is a mismatch and cannot work together.

Maybe pointing the graphiql endpoint to `grapple_urls.root `is more conform?

### PR DESCRIPTION
**Pull request by ruisaraiva19**
- Closes #192

### CODE REVIEW

File: grapple/urls.py [hunk 1]
```diff
@@ -24,7 +25,9 @@ def graphiql(request):
 
 
 # Traditional URL routing
-urlpatterns = [url(r"^graphql", csrf_exempt(GraphQLView.as_view()))]
+urlpatterns = [
```
Comment (zerolab) [inline grapple/urls.py:28]: suggestion: time to switch to `path`

otherwise, LGTM

Comment (ruisaraiva19) [reply grapple/urls.py:27]: Updated!

File: grapple/urls.py [hunk 2]
```diff
@@ -26,8 +25,8 @@ def graphiql(request):
 
 # Traditional URL routing
 urlpatterns = [
-    url(r"^graphql", csrf_exempt(GraphQLView.as_view()), name="grapple_graphql")
+    re_path(r"^graphql", csrf_exempt(GraphQLView.as_view()), name="grapple_graphql")
 ]
 
 if grapple_settings.EXPOSE_GRAPHIQL:
-    urlpatterns.append(url(r"^graphiql", graphiql, name="grapple_graphiql"))
+    urlpatterns.append(re_path(r"^graphiql", graphiql, name="grapple_graphiql"))
```
Comment (zerolab) [inline grapple/urls.py:32]: it should not need `re_path`, `path("graphiql")` and `path("graphql")` should do it

Comment (ruisaraiva19) [reply grapple/urls.py]: Updated!

**Review by zerolab — APPROVED** [review]

**Pull request merged**